feature-libs/organization/administration/core/store/effects/b2b-user.effect.ts

```diff
@@ -629,12 +640,34 @@ export class B2BUserEffects {
     )
   );
 
+  // TODO: make b2BUserCreationNotifierService a required dependency
```

Comment:
Please create a Jira ticket for it, and but the number into the code comment.
https://sap.github.io/spartacus-docs/breaking-changes/#adding-new-constructor-dependencies-in-minor-versions

Reply:
Changed